modules/javafx.controls/src/main/java/javafx/scene/control/skin/TreeTableRowSkin.java

```diff
@@ -332,20 +296,16 @@ public final DoubleProperty indentProperty() {
     /** {@inheritDoc} */
     @Override protected ObjectProperty<Node> graphicProperty() {
         TreeTableRow<T> treeTableRow = getSkinnable();
-        if (treeTableRow == null) return null;
+        // FIXME: illegal access if skinnable is null
```

Comment:
What is the purpose of removing the null check, but leaving getSkinnable() in there?
Given the signature of the method, it seems that it shouldn't ever return `null` in any case.